webapp-django/crashstats/base/jinja2/crashstats_base.html

```diff
@@ -106,7 +148,9 @@ <h1>Product Navigation</h1>
         </ul>
     {% endblock %}
 
-        <div class="search">
+        <div class="nav-links">
+            <a href="{{ url('documentation:home') }}">Documentation</a>
+            |
```

Comment:
I forgot... I think we should have a "file a bug" kind of thing here. It should be much easier to file a bug against Socorro than it is now.

Having said that, I'm not sure how best to do that. Maybe add a "File a bug" here? Maybe there's a better place in the header? Maybe clearer/shorter phrasing?

Reply:
We don't have to figure that out in this PR. I could do it in a follow-up one. But I figured I'd mention it in case you had ideas.

Reply:
I added a link to the [Bugzilla form for the Socorro component](https://bugzilla.mozilla.org/enter_bug.cgi?product=Socorro):

<img width="353" alt="screen shot 2019-01-14 at 3 37 53 pm" src="https://user-images.githubusercontent.com/193106/51148295-7e5cce00-1812-11e9-97c2-b5683c300531.png">